### PR TITLE
Debriefing tool/TER: operation start/end location & intended gear type missing checks

### PR DESCRIPTION
**Pull request by Mak-NOAA**
TERs: operation start/end location and intended gear type missing checks
Related to: https://github.com/nwfsc-fram/boatnet/issues/2151

### CODE REVIEW

File: util/tripChecks.ts [hunk 1]
```diff
@@ -445,4 +455,95 @@ function runFishProcessedCheck(tripErrorDoc: WcgopTripError, trip:any) {
...
+    //find the location with the min position in all locations
+    var minPositionedLocation = operation.locations.reduce(function(prev, current) {
+        if (+current.position < +prev.position) {
+            return current;
+        } else {
+            return prev;
+        }
+    });
+
+    if ( (trip.program.description === "TIQ Catch Shares" || //program_id in (14, 17)
```
[the remark this answers is not in the document]
Comment (Mak-NOAA) [reply util/tripChecks.ts:482]: It's TIQ in CouchDB, see all-programs document (key 14)

File: util/tripChecks.ts [hunk 2]
```diff
@@ -445,4 +455,95 @@ function runFishProcessedCheck(tripErrorDoc: WcgopTripError, trip:any) {
...
+    else if( latitude > 34.45 && latitude < 36.0)
+        returnArea = 300;
+    else if( latitude < 34.45 )
+        returnArea = 400;    
+
+    return returnArea;
+}
+
+//trip check code 106700 
+function runIntendedGearTypeMissingCheck(tripErrorDoc: WcgopTripError, trip: any) {
```
[the remark this answers is not in the document]
Comment (Mak-NOAA) [reply util/tripChecks.ts:524]: @sethgerou-noaa  they are blank at first and the observer or debriefer can set them on the Trips detail page

[the remark this answers is not in the document]
Comment (sethgerou-noaa) [reply util/tripChecks.ts:524]: ah, ok, thanks

File: util/tripChecks.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (sethgerou-noaa) [reply util/tripChecks.ts:482]: ok, I see that now - doesn't look like it was done in error.  I mentioned as a few fisheries for 2022 will be ITQ.  all good.

**Pull request merged**